langchain/client/runner_utils.py

```diff
@@ -342,8 +342,8 @@ async def arun_on_examples(
         llm_or_chain_factory: Language model or Chain constructor to run
             over the dataset. The Chain constructor is used to permit
             independent calls on each example without carrying over state.
-        concurrency_level: The number of async tasks to run concurrently.
-        num_repetitions: Number of times to run the model on each example.
+        concurrency_level: [Default: 5] The number of async tasks to run concurrently.
```

Comment:
no need to note defaults in docstrings i think, that's already made clear in function type hints

Reply:
- [+]